### PR TITLE
#3441: Add support for "TestCodeunitRangeFilter" field

### PR DESCRIPTION
**Pull request by hemisphera**
#3441
Adds new parameter "testCodeunitRange" to `Get-TestFromBcContainer` and `Run-TestsInBcContainer` that allows filling the test suite by using a codeunit-filter and using the TestCodeunitRangeFilter field already present on the command line test tool.

### CODE REVIEW

**Comment by hemisphera** [comment]
Just wondering ... there hasn't been any movement or comment on the PR for almost a month. Is there anything I am missing or do I just have to wait ...?

**Comment by freddydk** [comment]
> Just wondering ... there hasn't been any movement or comment on the PR for almost a month. Is there anything I am missing or do I just have to wait ...?

Just haven't had the time

File: AppHandling/Run-TestsInNavContainer.ps1 [hunk 1]
```diff
@@ -99,6 +106,8 @@ function Run-TestsInBcContainer {
         [Parameter(Mandatory=$false)]
         [string] $testCodeunit = "*",
         [Parameter(Mandatory=$false)]
+        [string] $testCodeunitRange = "*",
```
Comment (freddydk) [inline AppHandling/Run-TestsInNavContainer.ps1:109]: Default should not be "*" - default should be empty or null to make sure that the code doesn't run and won't change any behavior from earlier.

**Review by freddydk — CHANGES_REQUESTED** [review]
Please fix the review comment and then please add a line in the release notes, then I will approve.

File: AppHandling/Run-TestsInNavContainer.ps1 (hunk 1 above)
Comment (hemisphera) [reply AppHandling/Run-TestsInNavContainer.ps1]: Done.

**Comment by hemisphera** [comment]
> Please fix the review comment and then please add a line in the release notes, then I will approve.

Should be resolved.

**Review by freddydk — APPROVED** [review]

**Pull request merged**